zenodo/modules/records/serializers/schemas/dc.py

```diff
@@ -30,6 +30,25 @@
 
 from ...models import ObjectType
 
+from html.parser import HTMLParser
+
+
+class HTMLStripper(HTMLParser):
+    def __init__(self):
+        self.reset()
+        self.strict = False
+        self.convert_charrefs= True
+        self.fed = []
+    def handle_data(self, d):
+        self.fed.append(d)
+    def get_data(self):
+        return ''.join(self.fed)
+
+def strip_tags(html):
```

Comment:
You need to document your functions with docstrings, take a look at travis: https://travis-ci.org/zenodo/zenodo/jobs/211917811

Run ``./run_tests.sh`` locally.